apps/music/js/utils.js

```diff
@@ -75,13 +77,13 @@ function createListElement(option, data, index, highlight) {
       a.dataset.keyRange = 'all';
       a.dataset.option = data.option;
 
-      li.appendChild(titleSpan);
+      a.appendChild(titleSpan);
```

Comment:
Currently, the label (`<span>`) and link (`<a>`) are siblings. The screen reader "sees" the labels, but they are inoperable because the listener expects the link to be the target. When the user swipes through the list items they hear "label foo", and then on the next item simply "link". This needs to be cleaned up and made logical. So in this patch we make the labels proper children of the link elements to give these lists a logical markup and allow the screen reader to navigate and operate it correctly.